resource/less/colors.less

```diff
@@ -1,7 +1,15 @@
 
+// openmrs colors
+@omrs-red: #f26622;
```

Comment:
...well orange it is...

And the 'omrs-orange' shall be gold  as well.

Damned, I know these could be wrong...